public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -887,10 +887,12 @@ export class LokiDatasource
     // We want to interpolate these variables on backend
     const { __interval, __interval_ms, ...rest } = scopedVars;
 
+    const exprWithAdHoc = this.addAdHocFilters(target.expr);
+
```

Comment:
Is it worth adding a small test for this?

Something like 
```ts
  describe('applyTemplateVariables', () => {
    let ds: LokiDatasource;
    beforeEach(() => {
      ds = createLokiDSForTests();
    });

    it('should add the adhoc filter to the query', () => {
      const addAdHocFilterSpy = jest.spyOn(ds, 'addAdHocFilters');
      ds.applyTemplateVariables(
        { expr: 'testExpr', legendFormat: '' } as unknown as LokiQuery,
        { __interval: 1, __interval_ms: 10, testVar: 'testValue' } as unknown as ScopedVar
      );

      expect(addAdHocFilterSpy).toHaveBeenCalledWith('testExpr');
    });
  });
```

Reply:
thanks, good point! i added a version of this as a test 👍